polaris-service/build.gradle.kts

```
@@ -247,11 +247,16 @@ val startScripts =
     classpath = files(provider { shadowJar.get().archiveFileName })
   }
 
-tasks.register<Sync>("prepareDockerDist") {
-  into(project.layout.buildDirectory.dir("docker-dist"))
-  from(startScripts) { into("bin") }
-  from(shadowJar) { into("lib") }
-  doFirst { delete(project.layout.buildDirectory.dir("regtest-dist")) }
-}
+val prepareDockerDist =
+  tasks.register<Sync>("prepareDockerDist") {
+    into(project.layout.buildDirectory.dir("docker-dist"))
+    from(startScripts) { into("bin") }
+    from(shadowJar) { into("lib") }
+    doFirst { delete(project.layout.buildDirectory.dir("regtest-dist")) }
```

```suggestion
    doFirst { delete(project.layout.buildDirectory.dir("docker-dist")) }
```
However, the delete shouldn't be necessary, singe it's a `Sync` task. Can you verify this and if that's true remove this `doFirst`?

---

Will do (although in this PR, the change was only formatting :) )